README.md

```diff
@@ -1 +1,21 @@
-# Create React Admin Amplify App
\ No newline at end of file
+# Create React Admin Amplify App
+
+## iteration 1 manual steps
+
+- `npx create-react-app <name> <args>`
+- ~~detect yarn here and use it~~ do this later
+- `npm install react-admin`
+- ~~detect `amplify cli` or `aws cli`~~
+- ~~detect that `aws configure` has been run~~
+- detect that `@aws-amplify/cli` is installed
+- detect that `amplify configure` has been run [see](https://docs.amplify.aws/cli/start/install#option-1-watch-the-video-guide)
+- `amplify init`
+- `npm install @aws-amplify/core @aws-amplify/api @aws-amplify/auth`
+- `npm install react-admin-amplify`
+- `amplify add api`
+- either run `amplify add codegen` or `amplify push` to generate graphql files
```

Comment:
What's the difference? I hadn't come across codegen yet, I did some quick googling and but still uncear

Reply:
we need the graphql files and they are only generated my amplify from `codegen` or from a `push`. Push will create cloud resources so I think codegen is probably easier to use and definitely faster

Reply:
Okay, we'll still have an `amplify push` at some point though right?

Reply:
Yes

Reply:
Okay got it, thanks!